spec/lib/guard/puma_spec.rb

```diff
@@ -142,9 +149,13 @@
     end
 
     context "with custom :notifications option" do
-      let(:options) { { notifications: [:restarted] } }
+      let(:options) do
```

Comment:
I'm a fan of the [Semantic Rule](https://github.com/bbatsov/ruby-style-guide/issues/162) so let's change to `{}` instead of `do/end`.

Reply:
OK, no problem.